torch/_refs/__init__.py

```diff
@@ -918,6 +919,47 @@ def trunc(a):
     return prims.trunc(a)
 
 
+# TODO: register this as a real ref/decomposition once TorchInductor supports complex!
```

Comment:
It should be fine to register the decomposition and inductor won't try to use it by default.

Reply:
When I registered this as a decomposition, tests started failing and it seems Inductor generated code with `complex64` and like.

Reply:
Oh it is being picked up because you have it in the `core_aten_decompositions` list which is used by inductor here:
https://github.com/pytorch/pytorch/blob/78810d78e82f8e18dbc1c049a2b92e559ab567b2/torch/_inductor/decomposition.py#L64

Reply:
The function is composite(explicit) and it's already symintified in core, so there's not much point in registering it as a decomposition, not now nor when we support complex.

Reply:
@lezcano `CompositeExplicitAutograd` operators are not traced through by `FakeTensor`, so you still need a decomposition.

Reply:
Any reason why we wouldn't want to trace through those? Just because we are assuming that there may be some in-place ops or smth?

Reply:
Well in the case of `view.dtype` it calls raw `TensorImpl` methods and wouldn't materialize anything meaningful in the fx graph if we traced through it. Although I would question if that should be legal in composite methods. It certainly wouldn't work for any tensor subclasses so maybe should be registered as `CPU, CUDA, Meta`. 

Either way I'm guessing this function isn't alone in being registered as `CompositeExplicit` but actually doesn't dispatch to anything.